logstash-core-event-java/src/main/java/com/logstash/Event.java

```diff
@@ -250,4 +248,8 @@ public void tag(String tag) {
             tags.add(tag);
         }
     }
+
+    public static void setLogger(Logger logger) {
+        Event.logger = (logger == null) ? DEFAULT_LOGGER : logger;
```

Comment:
that's in line with @talevy comment above for the same thing but in the JRubu wrapper. I need to think about it.